docs/src/main/sphinx/connector/phoenix.md

```diff
@@ -50,6 +50,7 @@ The following Phoenix-specific configuration properties are available:
 | `phoenix.connection-url`      | Yes      | `jdbc:phoenix[:zk_quorum][:zk_port][:zk_hbase_path]`. The `zk_quorum` is a comma separated list of ZooKeeper servers. The `zk_port` is the ZooKeeper port. The `zk_hbase_path` is the HBase root znode path, that is configurable using `hbase-site.xml`.  By default the location is `/hbase`                                                                                                   |
 | `phoenix.config.resources`    | No       | Comma-separated list of configuration files (e.g. `hbase-site.xml`) to use for connection properties.  These files must exist on the machines running Trino.                                                                                                                                                                                                                                     |
 | `phoenix.max-scans-per-split` | No       | Maximum number of HBase scans that will be performed in a single split. Default is 20. Lower values will lead to more splits in Trino. Can also be set via session propery `max_scans_per_split`. For details see: [https://phoenix.apache.org/update_statistics.html](https://phoenix.apache.org/update_statistics.html). (This setting has no effect when guideposts are disabled in Phoenix.) |
+| `phoenix.scan-page-size`      | No       | The time limit on the amount of work single RPC request can do before it times out.                                                                                                                                                                                                                                                                                                              |
```

Comment:
> Phoenix Connector: Configure Page size while creating Connection

Please follow commit message guideline: https://trino.io/development/process#pull-request-and-commit-guidelines-